src/pieces/counting.ts

```diff
@@ -0,0 +1,127 @@
+import { Client, MessageEmbed, TextChannel } from 'discord.js';
+import { DB, CHANNELS } from '@root/config';
+import { Counter } from '../lib/types/Counter';
+
+let countingDB: Counter;
+let countingChannel;
+
+async function register(bot: Client): Promise<void> {
+	countingDB = await bot.mongo.collection(DB.COUNTING).findOne({ piece: 'counting' });
+	countingChannel = bot.channels.cache.get(CHANNELS.COUNTING_CHANNEL) as TextChannel;
```

Comment:
This might fail. You should probably fetch the channel instead of relying on the cache.